.circleci/src/jobs/upload_packages_to_server.yml

```diff
@@ -1,43 +0,0 @@
-docker:
-  - image: cimg/node:20.18.0
-parameters:
-  production:
-    description: Whether builds are delivered to production or beta.
-    type: boolean
-  bundle:
-    description: Defines which bundle should be deployed to the server.
-    enum: ["administration" , "backend"]
-    type: enum
-steps:
-  - prepare_workspace
-  - add_ssh_keys:
-      fingerprints:
-        - a1:3f:a7:c3:ff:12:40:1d:85:de:a7:ab:12:3f:cc:05
-  - when:
-      condition: << parameters.production >>
-      steps:
-        - run:
-            name: SFTP upload package
-            command: |
-              mkdir -p ~/.ssh
-              echo ${APT_FINGERPRINT_PRODUCTION} >> ~/.ssh/known_hosts
-              echo "Uploading: " ~/attached_workspace/debs/<< parameters.bundle >>/*.deb
-              sftp -b - ci@entitlementcard.tuerantuer.org:/srv/local-apt-repository/ \<<< "put -r ~/attached_workspace/debs/<< parameters.bundle >>/*.deb"
-        - run:
-            name: SFTP upload package
-            command: |
-              mkdir -p ~/.ssh
-              echo ${APT_FINGERPRINT_PRODUCTION} >> ~/.ssh/known_hosts
-              echo "Uploading: " ~/attached_workspace/debs/<< parameters.bundle >>/*.deb
-              sftp -b - ci@entitlementcard.tuerantuer.org:/srv/local-apt-repository/ \<<< "put -r ~/attached_workspace/debs/<< parameters.bundle >>/*.deb"
```

Comment:
@f1sh1918 do you know if there was a specific reason why this was executed twice?

Reply:
Okay, lets just try with a beta delivery once the current beta is promoted?

Reply:
Seems to work fine without. `~` is not working though, needs to be `/home/circleci`.